tests/system/action/organization/test_initial_import.py

```diff
@@ -32,15 +33,28 @@ def test_initial_import_with_initial_data_file(self) -> None:
     def test_initial_import_with_example_data_file(self) -> None:
         self.datastore.truncate_db()
         request_data = {"data": get_initial_data_file("global/data/example-data.json")}
+        request_data["data"]["organization"]["1"]["default_language"] = "de"
         response = self.request("organization.initial_import", request_data)
         self.assert_status_code(response, 200)
+        Translator.set_translation_language("de")
         for collection in request_data["data"]:
             if collection == "_migration_index":
                 continue
             for id_ in request_data["data"][collection]:
-                self.assert_model_exists(
-                    f"{collection}/{id_}", request_data["data"][collection][id_]
-                )
+                entry = request_data["data"][collection][id_]
+                if collection == "organization":
+                    for field in (
+                        "login_text",
+                        "legal_notice",
+                        "users_email_subject",
+                        "users_email_body",
+                    ):
+                        if entry.get(field):
+                            entry[field] = Translator.translate(entry[field])
+                if collection == "theme":
+                    if entry.get("name"):
+                        entry["name"] = Translator.translate(entry["name"])
+                self.assert_model_exists(f"{collection}/{id_}", entry)
 
```

Comment:
add a test where you don't provide a `default_language`

Reply:
Add a test with missing default_language.